src/main/resources/assets/js/app/inputtype/text/HtmlArea.ts

```diff
@@ -64,13 +68,38 @@ export class HtmlArea
         this.inputConfig = config.inputConfig;
 
         this.authRequest =
-            new IsAuthenticatedRequest().sendAndParse().then((loginResult: LoginResult) => {
-                this.editableSourceCode = loginResult.isContentExpert();
-            });
+            Q.all([new IsAuthenticatedRequest().sendAndParse(), new ProjectGetRequest(ProjectContext.get().getProject()).sendAndParse()])
```

Comment:
This has to be changed so that `ProjectGetRequest` is sent only if current user is not either an admin or cms.expert.

Reply:
Ok